.scalafmt.conf

```diff
@@ -22,3 +22,11 @@ project.git = true
 
 runner.dialect = scala213
 
+project.excludeFilters = [
+   # can't get fileOverride to work to specify scala3 as dialect for these
+  "scalalib/test/resources/hello-dotty/*"
+  # these have scaladoc comments with a particular shape that we don't want to reformat
+  "example/depth/large/*"
```

Comment:
The only reason we don't need to ignore the rest of `example/` is because Scalafmt doesn't pick up the `.mill` files. If Scalafmt learns how to do so, or if we rename them to `.mill.scala`, we'll need to go back and ignore the rest of `example/` as well